src/main/java/Registration.java

```diff
@@ -0,0 +1,35 @@
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet (name = "servlet", urlPatterns = "/servlet")
+public class Registration extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
+       getServletContext().getRequestDispatcher("/webapp/save-request.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String name = req.getParameter("name");
```

Comment:
In you form on jsp, you have `nameInput` and `emailnput`. Here you should also use these arguments (since we retrieve values by the input tag `name` attrinute). Try to do it and fix another comment.
Hope, it will help